pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,115 @@
+// ------------------------------
+// edX Pattern Library: Utilities - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$alert-border-radius: rem(3);
+// border around the sides and bottom
+$alert-border: rem(1) solid palette(grayscale-cool, light);
+
+// alert colors
+$alert-information-color: palette(information, base);
+$alert-warning-color: palette(warning, base);
+$alert-error-color: palette(error, base);
+$alert-success-color: palette(success, base);
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+@mixin alert($alert-color) {
+    border-top:  rem(4) solid $alert-color;
+
+    .alert-icon {
+        color: palette(grayscale, white);
+        background-color: $alert-color;
+    }
+}
+
+// ----------------------------
+// #GENERAL
+// ----------------------------
+.alert {
+    border: $alert-border;
```

Comment:
@dsjen, one thing I'm not seeing here is a ``background-color`` property. You may want to add that (and the appropriate Sass configuration variable, for theming purposes). I think the Doc Site's rendering might be giving you that white background for free in our previews, but that needs to be provided for real use. Make sense?